src/app/components/client/ExposureCard.tsx

```diff
@@ -438,15 +438,15 @@ const SubscriberBreachCard = (props: SubscriberBreachCardProps) => {
                   },
                 }
               )}
-              <Link href={`/breach-details/${subscriberBreach.name}`}>
-                <span>
+              <a href={`/breach-details/${subscriberBreach.name}`}>
```

Comment:
I don't quite get why this needed to be turned into a plain `<a>`? Isn't it still a relative link?